### ISSUE TITLE
[1.15.2 Pre SHADER] Invisible chunks caused by other mod (Low Priority)

### ISSUE THREAD

**Issue by gcjonata** [sp614x/optifine#3893]
## Description of Issue
I in a modded server, using this mods, and i get the same bug using the BSL shader and the TME Shader, on SEUS Shader i don't get this bug.

## Steps to Reproduce
1. Load the world and walk a fell chunks

## OptiFine Version
1.15.2 Pre

## Installation Method
just droop the .jar on mod folder
obs: OptFine are not installed on the server

## Fabric/Forge Version
Forge 31.1.63

## Other Installed Mods
AlchemyPath-1.0.0.jar (private mod)
amb-1.15.2-0.5.1.jar
anviltweaks-2.2a.jar
backpacked-1.4.1-1.15.2.jar
BiomesOPlenty-1.15.2-10.0.0.345-universal.jar
Blur-1.1.0-1.jar
BonsaiTrees-2.1.2.6.jar
Clumps-5.0.2.jar
extrabows-v1.15.2+b1.jar
FastLeafDecay-v20.jar
Hwyla-forge-1.10.8-B72_1.15.2.jar
inventorysorter-1.15.2-17.0.0.jar
jei-1.15.2-6.0.0.2.jar
journeymap-1.15.2-5.7.0beta1.jar
libnonymous-1.15.2-1.1.1.5.jar
MixinBootstrap-1.0.2.jar
MouseTweaks-2.13-mc1.15.1.jar
OreExcavation-1.7.151.jar
OresAboveDiamonds+1.15+v3.3.jar
phosphor-forge-mc1.15.2-0.5.2+build.4.jar
Placebo-1.15.2-3.0.2.jar
preview_OptiFine_1.15.2_HD_U_G1_pre16.jar
swingthroughgrass-1.15.2-1.4.0.jar

## F3 Debug Screenshot
![image](https://user-images.githubusercontent.com/47560420/81331127-7f54e700-9077-11ea-888e-3cd7a699cad4.png)

If you've installed using Forge/Fabric:
- Does this happen without other mods installed? 
   - Only OptiFine and Forge
- Does this happen without any mods at all?
   - No.
- Have you tried using a binary search to find which mods cause this issue?
   - phosphor-forge-mc1.15.2-0.5.2+build.4.jar this mod are causing the bug

**Comment by Mig979** [comment]
I have this issue as well, even without using shaders it happens. Removing Phosphor or Optifine makes it work properly, either of them work fine on their own.

**Comment by Ivan0xFF** [comment]
I'm having this issue as well, but not with Phosphor. JourneyMap also causes this to happen, they have their own issue opened here: https://github.com/TeamJM/journeymap/issues/220

**Comment by gcjonata** [comment]
i open my game today and this bug back, i will try to remove the JourneyMap
![image](https://user-images.githubusercontent.com/47560420/81477053-e1365d80-91eb-11ea-9a04-4fe76eb11892.png)

**Comment by anarisis** [comment]
Can confirm this issue crops up with latest version of JourneyMap and without Phosphore. F3+A renders the invisible chunks, but rendering will stop again at the edge of the render distance.

**Comment by sp614x** [comment]
This is a known problem with JourneyMap.

**Comment by HireSLH** [comment]
Still happens on Pre 22

**Comment by mateo122597** [comment]
having the same issue with pre 22, last version forge and journey map... someone managed to solve this? or is there a good replacement for journey map ?

**Comment by HireSLH** [comment]
> having the same issue with pre 22, last version forge and journey map... someone managed to solve this? or is there a good replacement for journey map ?

F3+A

**Comment by mysticdrew** [comment]
> This is a known problem with JourneyMap.

I am open to ideas how to resolve this on my end, but I have not found a solution.

**Comment by naeqirelle2020** [comment]
afaik there isn't one that i've seen aside from removing it

**Comment by mysticdrew** [comment]
> afaik there isn't one that i've seen aside from removing it

I mean in code, I'm the journeymap dev

**Comment by naeqirelle2020** [comment]
oh i'm not sure

**Comment by mateo122597** [comment]
> > having the same issue with pre 22, last version forge and journey map... someone managed to solve this? or is there a good replacement for journey map ?
> 
> F3+A

Yeah i know, got tired of pressing those all the time so i removed it. I'm using xaeros now

**Comment by EliTheMonkeyGuy** [comment]
I managed to fix the issue on my end with both Journey Map and Optifine installed. All I did was disable way-point beacons, turn off rotating beacons, and then re-enable way-points beacons. I don't know whether this was a coincidence or if it is temporary, but I suggest you give it a shot.
